### PR TITLE
workflow: clean up (fix #130)

### LINKED ISSUE #130: Too many duplicated workflow-related files
Looking at each package in `packages` directory, there are too many duplicated workflow-related files in each package, such as some test-related files.

Also, every `packages.json` contains many unused devDependencies.
For example, we just need to install `typescript` and `mocha` even other similar packages once in the root `package.json`, but these dependencies were specify in each `package.json`, which is redundant.

Not necessary: If there are some shared testing-related utils or functions, I recommend to use 3rd-party assertion library, or even use Jest which provides full features out of the box.

/cc @fkleuver

### PR DESCRIPTION
**Pull request by g-plane**
Because of using Yarn's workspaces, there is no need to specify same dependencies many times.

### CODE REVIEW

File: package-scripts.js [hunk 1]
```diff
@@ -5,11 +5,11 @@ function config(name) {
 }
 
 function rollup(mod, minify) {
-  return package(`rollup --c --environment mod:${mod},${minify ? 'minify' : ''}`);
+  return `rollup --c --environment mod:${mod},${minify ? 'minify' : ''}`;
```
Comment (fkleuver) [inline package-scripts.js]: I'd rather not remove the package wrapper. It ensures that the scripts can run without the need to have these tools globally installed, and that they run cross-platform. I see no direct benefit in removing this, only risks.

File: package-scripts.js [hunk 2]
```diff
@@ -5,11 +5,11 @@ function config(name) {
...
 }
 
 function mocha(arg) {
-  return crossEnv(`TS_NODE_PROJECT=\'${config('test')}\' ${package('mocha')} ${arg}`);
+  return crossEnv(`TS_NODE_PROJECT=\'${config('test')}\' mocha ${arg}`);
```
Comment (fkleuver) [inline package-scripts.js]: Ditto

File: packages/cherow/.travis.yml [hunk 3]
```diff
@@ -1,19 +0,0 @@
-language: node_js
```
Comment (fkleuver) [inline packages/cherow/.travis.yml]: Why are you removing this file?

File: package-scripts.js (hunk 1 above)
Comment (g-plane) [reply package-scripts.js]: This is not globally installed and it's located in the root node_modules directory. It's Ok so need to change it.

File: packages/cherow/.travis.yml (hunk 3 above)
Comment (g-plane) [reply packages/cherow/.travis.yml]: It's redundant. There is an existing Travis CI config file in root.

File: package-scripts.js (hunk 1 above)
Comment (g-plane) [reply package-scripts.js]: You can try by yourself: Create a new directory, install some packages which contain CLI command and add it to `package.json` without `./node_modules/.bin/` prefix, you will find everything works.

File: packages/cherow/.travis.yml (hunk 3 above)
Comment (fkleuver) [reply packages/cherow/.travis.yml]: Right. Fair enough :)

File: package-scripts.js (hunk 1 above)
Comment (fkleuver) [reply package-scripts.js]: If you're sure all scripts still work without the packages globally installed then it's ok I guess :)

Comment (g-plane) [reply package-scripts.js]: TBH, I haven't installed any packages globally on my computer.

**Pull request merged**